### PR TITLE
Fix(bug): about next over func

### PR DESCRIPTION
**Pull request by yuanlianghu**

### CODE REVIEW

**Comment by yuanlianghu** [comment]
@kurikomoe hahaha

**Comment by kurikomoe** [comment]
@yuanlianghu  ？？？？

File: gdb/infrun.c [hunk 1]
```diff
@@ -6701,10 +6701,35 @@ process_event_stop_test (struct execution_control_state *ecs)
...
+      res = 0;
+      cngdb_abi_resume (1, dev, cluster, core);
+      while (!res)
+        {
+          cngdb_api_query_stop (&res);
+        }
+    }
   if (!frame_id_eq (get_stack_frame_id (frame),
                     ecs->event_thread->control.step_stack_frame_id)
-      && ((cngdb_focus_on_device() && !cngdb_on_start() && stop_pc >= 5 || frame_id_eq (frame_unwind_caller_id (get_current_frame ()),
```
Comment (kurikomoe) [inline gdb/infrun.c:6706]: > stop_pc >= 5

please use the macro definition rather than raw number.

**Comment by yuanlianghu** [comment]
@kurikomoe review the merge requests

**Pull request merged**